test/Dapr.Client.Test/InvocationHandlerTests.cs

```diff
@@ -148,6 +153,55 @@ public async Task SendAsync_RewritesUri_AndAddsApiToken()
             Assert.False(request.Headers.TryGetValues("dapr-api-token", out _));
         }
 
+        [Fact]
+        public async Task SendAsync_RewritesUri_AndAddsApiToken_WithEnvVarSet()
+        {
+            Environment.SetEnvironmentVariable(Constants.DaprApiTokenEnvironmentVariable, "super-duper-secure");
+            var uri = "http://bank/accounts/17?";
+
+            var capture = new CaptureHandler();
+            var handler = new InvocationHandler()
+            {
+                InnerHandler = capture,
+
+                DaprEndpoint = "https://localhost:5000",
+            };
+
+            var request = new HttpRequestMessage(HttpMethod.Post, uri);
+            var response = await CallSendAsync(handler, request);
+
+            Assert.Equal("https://localhost:5000/v1.0/invoke/bank/method/accounts/17?", capture.RequestUri?.OriginalString);
+            Assert.Equal("super-duper-secure", capture.DaprApiToken);
+
+            Assert.Equal(uri, request.RequestUri?.OriginalString);
+            Assert.False(request.Headers.TryGetValues("dapr-api-token", out _));
+        }
+
+        [Fact]
+        public async Task SendAsync_RewritesUri_WithApiTokenAndEnvVarSet_EnvVarIsIgnored()
+        {
+            Environment.SetEnvironmentVariable(Constants.DaprApiTokenEnvironmentVariable, "super-duper-secure2");
```

Comment:
Again, my feedback here is that this is a test for code that *we don't have*. Nothing in the InvocationHandler reads this environment variable, and we wouldn't add it back because we have no reason to.